src/index.js

```diff
@@ -1,15 +1,15 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+/* eslint-disable */
+import React from "react";
+import ReactDOM from "react-dom";
+import { HashRouter  } from "react-router-dom";
+import TodoContainer from "/home/isaka/to-do/src/functionalBased/components/TodoContainer.js";
+import "/home/isaka/to-do/src/functionalBased/App.css";
```

Comment:
You did a great job importing files at index file but kindly fix the path of TodoContainer and CSS file because the app is not opening and it is showing an error.

Reply:
HI @Donard97👋 , thanks for the suggestion i fixed this issue already